src/java/org/apache/cassandra/transport/CBUtil.java

```diff
@@ -162,6 +166,8 @@ public static void writeAsciiString(String str, ByteBuf cb)
     public static void writeString(String str, ByteBuf cb)
     {
         int length = TypeSizes.encodedUTF8Length(str);
+        Preconditions.checkArgument(length <= Short.MAX_VALUE,
```

Comment:
this didn't happen but was a concern I had when trying to debug...

Reply:
this is going to be trunk only, to make sure i don't make released branches unstable i left this out of the backports